### PR TITLE
Allow multiple S1 processes

### PR DESCRIPTION
**Pull request by nidico**
* Don't use hardcoded `s1_process_url` for S1 header buttons
* Show meaningful process information in header and landing page

Note that this makes use of the header slot concept, so that other processes can also inject their `ProcessHeaderSlot` into the S1 platform header.

### CODE REVIEW

File: src/s1/s1/static/js/Packages/S1/Proposal/Proposal.ts [hunk 1]
```diff
@@ -306,7 +306,7 @@ export var listingDirective = (
         },
         link: (scope) => {
             scope.contentType = RIProposalVersion.content_type;
-            scope.$on("$destroy", adhTopLevelState.bind("processUrl", scope));
+            scope.$on("$destroy", adhTopLevelState.bind("processUrl", scope)); // REFACT is this even neccessary?
```
Comment (der-john) [inline src/s1/s1/static/js/Packages/S1/Proposal/Proposal.ts]: yes. It's used in `Listing` to get list elements (=proposals).

Comment (dwt) [inline src/s1/s1/static/js/Packages/S1/Proposal/Proposal.ts]: Feel free to remove the comment then. :)

**Comment by dwt** [comment]
When testing, please note that the s1-process should have a title, short and long description (editable via the SDI) which should be displayed on the landing page and the header.

**Comment by der-john** [comment]
looks good & works.

**Comment by dwt** [comment]
thanks

**Pull request merged**